### ISSUE TITLE
script errors out after acknowledging "spreakers will not work"

### ISSUE THREAD

**Issue by g1001** [WeirdTreeThing/chromebook-linux-audio#168]
**Describe the bug**
Running on Samsung Chromebook Plus V2 with Fedora Kinoite installed (after following chrultrabook procedure).

Here is the output of the script:
WARNING: You may run into audio issues, even after running this script. Please report any issues on githu
b.
Installing AVS
WARNING: Your device has max98357a and can cause permanent damage to your speakers if you set the volume
too loud! As a safety precaution devices with max98357a have speakers disabled until a fix is in place. H
eadphones and HDMI audio are safe from this.
If you want to disable this check, restart the script with --force-avs-install
Type "I Understand my speakers will not work since my device has max98357a!" in all caps to continue: I U
NDERSTAND MY SPEAKERS WILL NOT WORK SINCE MY DEVICE HAS MAX98357A!
Installing topology
Traceback (most recent call last):
 File "/var/home/adele/chromebook-linux-audio/./setup-audio", line 319, in <module>
   match_platform(board)
 File "/var/home/adele/chromebook-linux-audio/./setup-audio", line 95, in match_platform
   avs_audio()
 File "/var/home/adele/chromebook-linux-audio/./setup-audio", line 151, in avs_audio
   urlretrieve(url=avs_tplg_url, filename="/tmp/avs_tplg/avs-tplg.tar.gz")
 File "/usr/lib64/python3.12/urllib/request.py", line 240, in urlretrieve
   with contextlib.closing(urlopen(url, data)) as fp:
                           ^^^^^^^^^^^^^^^^^^
 File "/usr/lib64/python3.12/urllib/request.py", line 215, in urlopen
   return opener.open(url, data, timeout)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 File "/usr/lib64/python3.12/urllib/request.py", line 521, in open
   response = meth(req, response)
              ^^^^^^^^^^^^^^^^^^^
 File "/usr/lib64/python3.12/urllib/request.py", line 630, in http_response
   response = self.parent.error(
              ^^^^^^^^^^^^^^^^^^
 File "/usr/lib64/python3.12/urllib/request.py", line 559, in error
   return self._call_chain(*args)
          ^^^^^^^^^^^^^^^^^^^^^^^
 File "/usr/lib64/python3.12/urllib/request.py", line 492, in _call_chain
   result = func(*args)
            ^^^^^^^^^^^
 File "/usr/lib64/python3.12/urllib/request.py", line 639, in http_error_default
   raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

**Boardname**
Nautilus

**Logs**
Thought the above was probably sufficient but can do this if needed.

**Comment by mavr1iq** [comment]
there is a pull request which is "fixing" this problem. but another problem pops up. i am pretty new to this so idk how to make it work, maybe you will try and tell

**Comment by g1001** [comment]
OK, now that my wife is off her laptop, I tried running the wget to get the logs, but it also had a problem.

Since this wget behavior seems to be a separate problem, I'm opening a new issue for it.

The log file produced is only 212 bytes, and its contents are:
no-alsautils  0 B
no-root         0 B
no-strace     0 B

**Comment by g1001** [comment]
Prior to koluckirafal's reply, I tried a bunch of things to get around this problem, including some commenting out of lines in the setup-audio script, but I eventually got to a point where the script produced a large number errors trying to write to Fedora Kinoite's read-only filesystem.
At that point, I gave up and switched to the more conventional Fedora Plasma Spin. 
The audio is now working. It appears that an update to Plasma Spin after the intial install was what did the trick.
So closing this issue, tho I can't say the problem was "solved". I think atomic distros like Kinoite lead to too many problems.
